apps/pie-docs/src/components/assistive-text/overview/overview.md

```diff
@@ -151,8 +151,6 @@ If a validation variant of the assistive text is required for error or success,
 
 ## Examples
 
-Outlines the atomic level interactive elements for the component.
```

Comment:
I don't expect you to address it in this PR, but I think basically all of the pages have this

Reply:
Hey Xander, you're right, i just realised this today. I think I will go through the docs (in figma first) tomorrow to delete this line under Examples. And then i will check the docsite as well.